garglk/CMakeLists.txt

```diff
@@ -166,20 +166,7 @@ if(WITH_TTS)
     if(APPLE)
         target_sources(garglk PRIVATE ttsmac.mm)
     elseif(UNIX)
-        find_package(PkgConfig REQUIRED)
-
-        if(NOT WITH_TTS STREQUAL "AUTO")
-            set(TTS_REQUIRED "REQUIRED")
-        endif()
-
-        pkg_check_modules(SPEECH_DISPATCHER ${TTS_REQUIRED} IMPORTED_TARGET speech-dispatcher)
-
-        if(SPEECH_DISPATCHER_FOUND)
-            target_sources(garglk PRIVATE ttsspeechd.cpp)
-            target_link_libraries(garglk PRIVATE PkgConfig::SPEECH_DISPATCHER)
-        else()
-            target_sources(garglk PRIVATE ttsnull.c)
-        endif()
+        target_sources(garglk PRIVATE ttsspeechd.cpp)
```

Comment:
For dlopen(), you'll also need to link against `${CMAKE_DL_LIBS}` for platforms which require a specific library for support (e.g. `libdl`).